src/create-storage.js

```diff
@@ -11,7 +11,7 @@ let isWorkerAvailable = window.Worker;
 
 let workify, worker;
 if (isWorkerAvailable) {
-	workify = require('webworkify');
+	workify = isWebpack ? require('webworkify-webpack') : require('webworkify');
```

Comment:
@dfcreative Do you mean a `browser` field in the user's `package.json` field? or in `gl-waveform`'s `package.json`